tests/unit/worker/worker_test.py

```diff
@@ -75,7 +60,7 @@ def example_test_task(x):
         mock.assert_called_with(job_key=job.key, variables=job.variables)
 
 
-def test_add_task():
+def test_add_task(zeebe_worker, task):
```

Comment:
This is defined twice (not introduced now, but renaming it will result in an additional test being run)